dbms/src/Flash/tests/gtest_join_executor.cpp

```diff
@@ -738,10 +768,11 @@ try
     ColumnsWithTypeAndName left_column_data;
     ColumnsWithTypeAndName right_column_data;
     ColumnsWithTypeAndName common_column_data;
-    size_t table_rows = 102400;
-    size_t common_rows = 20480;
+    size_t table_rows = 51200;
+    size_t common_rows = 10240;
     UInt64 max_block_size = 800;
-    size_t original_max_streams = 20;
+    size_t max_streams_big = 20;
+    size_t max_streams_small = 4;
```

Comment:
What is the difference between these two?
It looks like the following test doesn't respect this, using the table_concurrency_hint.

Reply:
`max_streams_small ` is used to test the case that fine_grained_stream_count bigger than max_stream, I will update the test to use `max_streams_small` only fine_grained_stream is enabled.